### ISSUE TITLE
No alt text for images on events page

### ISSUE THREAD

**Issue by jchristgit** [python-discord/site#1381]
As pointed out in #1363:

Many images on the events page lack a definition for alt text.

All images need alt text defined so that screen readers can present the information to the user. If an image is not necessary for a screen reader user, it should define alt="".

**Comment by DragonSenseiGuy** [comment]
@jb3 I'll take a look at this
